understanding/22/visible-controls.html

```diff
@@ -82,10 +82,9 @@ <h2>Benefits of Visible Controls</h2>
       <h2>Examples of Visible Controls</h2>
 
     <ul>
-         <li>Users with cognitive and learning disabilities can more easily locate controls that allow them to successfully progress through a process.</li>	         <li>Users with cognitive and learning disabilities, mobility and motor impairments, and low vision can more easily locate controls that allow them to successfully progress through a process.</li>
-         <li>Users with memory impairments do not need to remember where controls are located</li>	         <li>Users with memory impairments do not need to remember where controls are located.</li>
-        <li>Users with low vision do not need to combine visually searching for controls with moving pointer hover or keyboard focus.</li>
-        <li>Users with mobility and motor impairments who use speech input do not need to use hover or focus to locate and operate controls.</li>
+      <li>In a web editor, once a page is open for editing, all the possible locations for editing display a pencil icon.</li>
+      <li>In a web editor, a settings can be changed so that the icons are all shown by default. If the setting isn’t enabled, the possible locations for editing are hidden until a user hovers over them.</li>
+      <li>When viewing a list of emails the controls needed to delete or forward the email are not visible unless by hovered. When an email has been selected the same controls are persistently visible.</li>
```

Comment:
unless <del>by </del>hovered

Reply:
Typo, "a settings can be changed..." should be "a setting can be changed..."